helm-charts/konk/templates/certificate.yaml

```
@@ -0,0 +1,12 @@
+apiVersion: cert-manager.io/v1alpha2
+kind: Certificate
+metadata:
+  name: {{ include "konk.fullname" . }}-certificate
```

Can we use a name that describes its purpose a little better? Is this cert used for ingress support? Maybe this would be a better name:
```suggestion
  name: {{ include "konk.fullname" . }}-ingress-client
```